tests/helpers/load_state_tests.py

```diff
@@ -103,8 +102,12 @@ def add_block_to_chain(
     if not mock_pow:
         load.state_transition(chain, block)
     else:
-        with patch(
-            f"ethereum.{load.fork_module}.fork.validate_proof_of_work",
+        fork_module = importlib.import_module(
+            f"ethereum.{load.fork_module}.fork"
+        )
+        with patch.object(
+            fork_module,
+            "validate_proof_of_work",
```

Comment:
@gurukamath @petertdavies if you could double check that this change maintains the original behaviour, I'd appreciate it!